lib/common/math.ts

```diff
@@ -83,3 +83,10 @@ export function isSafeNumber(value: number): boolean {
     typeof value === 'number' && Number.isFinite(value) && !Number.isNaN(value)
   );
 }
+
+/**
+ * Converts a value in radians to degrees.
+ */
+export function rad2deg(rad) {
+  return rad * (180 / Math.PI);
+}
```

Comment:
sorry, i think I've been looking at too much plain js to catch this in my first review

```suggestion
export function rad2deg(rad: number): number {
  return rad * (180 / Math.PI);
}
```